tests/bfd/test_bfd_static_route.py

```diff
@@ -20,7 +22,7 @@ class TestBfdStaticRoute(BfdBase):
     TOTAL_ITERATIONS = 100
```

Comment:
Currently we have 100 iterations for BFD flap test, which will take a long time (~4h) to complete. I will change the iteration number to be based on the completeness level in my next PR (Microsoft ADO 28301940)